.cargo/config.toml

```diff
@@ -1,7 +1,7 @@
 [target.x86_64-pc-windows-msvc]
 rustflags = ["-Ctarget-feature=+crt-static"]
 [target.i686-pc-windows-msvc]
-rustflags = ["-Ctarget-feature=+crt-static"]
+rustflags = ["-C", "target-feature=+crt-static", "-C", "link-args=/NODEFAULTLIB:MSVCRT"]
```

Comment:
I have a question about this, I tried to do this by rustflags environemnt as https://github.com/Shylock-Hg/rustdesk/pull/11/files, but it failed. Is there something different?
@rustdesk